src/main/kotlin/com/sourcegraph/cody/config/SettingsMigration.kt

```diff
@@ -83,6 +85,33 @@ class SettingsMigration : Activity {
     migrateEnterpriseAccount(project, requestExecutorFactory, customRequestHeaders)
   }
 
+  private fun migrateLlms(project: Project) {
+    HistoryService.getInstance(project)
+        .state
+        .chats
+        .filter { it.llm == null }
+        .forEach {
+          val (provider, title) =
+              modelToProviderAndTitle.getOrDefault(it.model, Pair("Unknown", "Unknown"))
```

Comment:
I assume this is mostly used of displaying to user, right?
Then maybe instead of `Pair("Unknown", "Unknown")`:
```kotlin
val nameParts = it.model.split("/")
val provider = nameParts.firstOrNull() ?: "Unknown"
val model = nameParts.lastOrNull() ?: "Unknown"
Pair(provider, model)
```

It will be much more user friendly if some model will be unexpectedly added before we catch up in the code.

Reply:
I thought we are storing the models as `modelID` but that's not true. it was `displayName` 😕 I must map display names - pls retake a look here